app/controllers/mixins/ems_common_angular.rb

```diff
@@ -98,7 +98,7 @@ def render_validation_result(result, details)
         level = :error
       end
 
-      render_flash_json(msg, level)
+      render_flash_json(msg, level, :long_alert => true)
```

Comment:
Nowsg is unlimited, perhaps ruby should still truncate to some large limit here, to avoid sending megabyte responses to UI.

Reply:
Is this addressed by https://github.com/ManageIQ/manageiq/pull/16550/files#diff-f9cace1fe4fdcec66f962b23f2110fd6R337 now?